docs/grammar.txt

```diff
@@ -39,4 +39,4 @@ slice = "[" [ expression ] [ ":" expression ] "]";
 lambda = "lambda" [ lambda_arg { "," lambda_arg } ] ":" expression;
 lambda_arg = Ident [ "=" expression ];
 operator = ("+" | "-" | "%" | "<" | ">" | "and" | "or" | "is" | "is" "not" |
-            "in" | "not" "in" | "==" | "!=" | ">=" | "<=" | "|");
+            "in" | "not" "in" | "==" | "!=" | ">=" | "<=" | "|" | "*");
```

Comment:
Would it make more sense for this to be grouped with the other arithmetic operators?

Reply:
good point, done